src/lib/philips.ts

```diff
@@ -80,7 +80,11 @@ export function philipsLight(args?: modernExtend.LightArgs & {hueEffect?: boolea
             );
             result.configure.push(async (device, coordinatorEndpoint, definition) => {
                 for (const ep of device.endpoints) {
```

Comment:
```suggestion
                for (const ep of device.endpoints.filter((ep) => ep.supportsInputCluster('manuSpecificPhilips2'))) {
```

The issue here is that we try to bind ep 242 (GP endpoint) which doesn't support this cluster. Instead we should only bind eps which support this cluster. Try catch can be removed after this.